courses/fundamentals_of_ada/adv_245_ravenscar_tasking.rst

```diff
@@ -65,38 +116,39 @@ Ravenscar Tasking Limitations
 * Active synchronization not supported
 
    - Asymmetric rendezvous
-   - Entries
+   - Task entries
 
-* Tasks can only be declared at library level
+* Tasks declaration **must be** at library level
 
-   - All have to finish before the program terminates
+   - They must **never** finish
```

Comment:
This is opposite what it said before, but if you verified it, then OK

Reply:
Yes, it is a restriction of the ravenscar profile, but I believe it is not really enforced.

This means that technically if all tasks stopped, then the program would stop, but then it would not be a ravenscar program anymore.